src/datumaro/plugins/data_formats/coco/extractor_merger.py

```diff
@@ -0,0 +1,95 @@
+# Copyright (C) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+from collections import defaultdict
+from typing import Sequence
+
+from datumaro.components.contexts.importer import _ImportFail
+from datumaro.components.dataset_base import SubsetBase
+from datumaro.components.merge import ExactMerge
+from datumaro.components.merge.extractor_merger import ExtractorMerger, check_identicalness
+from datumaro.plugins.data_formats.coco.base import _CocoBase
+
+
+class COCOTaskMergedBase(SubsetBase):
```

Comment:
Such name suggests that there are going to be some derived classes, but I can't find such. Are there any derived classes in upstream?

Reply:
Apparently, they renamed "Extractor" and used "Base" instead.